### PR TITLE
Setup project

### PR DESCRIPTION
**Pull request by shaqayq**
In this part of the project, I created React app and also add Linters files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,73 +1,70 @@
...
 
-- Major languages
-- Frameworks
-- Technologies used
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
 
-## Live Demo (if available)
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
```
Comment (dasileker) [inline README.md]: - [ ] Please add relevant details to the readme file a how to clone and install packages and start the app.

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @shaqayq,** 

### **Good job so far!**
There are some issues that you still need to work on to go to the next project but you are almost there!

## **Highlights**
- Nice PR✅.
-  Nice start on adding the app✅.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (shaqayq) [reply README.md]: @dasileker 
Thanks for reviewing my project. I will do it.  😊

**Review by DammyShittu — APPROVED** [review]
Hi @shaqayq,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR
✔️ Passing linters

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**